### ISSUE TITLE
Label vertices to help with prose

### ISSUE THREAD

**Issue by manavkataria** [mapbox/vector-tiles#29]
This is a suggestion to improve the documentation not an error. 

Under section **What the spec doens't cover > Simplification** it would help if the vertices are explicitly labelled {A, B and C} and we use the labels to show their transition as the vertices snap to tile grid. Without the labels its confusing how the last vertex of the triangle gets flipped, if you just look at the GIF and not read the text. With labels, the GIF will be self-sufficient and can aid the prose as well.

![screen shot 2016-12-05 at 8 08 12 pm](https://cloud.githubusercontent.com/assets/2206789/20912777/1587bd6c-bb27-11e6-85d1-c486547de5e9.jpg)

**Comment by mapsam** [comment]
Good idea @manavkataria! I'll take a peak at this when I get a chance.
